### ISSUE TITLE
organize callback call to eg to inform that task is finished 

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/entrypoint#3]
related ticket is: https://github.com/zebrunner/esg/issues/387

**Comment by vdelendik** [comment]
changes are available in `entrypoint:2.2-beta1` image and might be tested on any edv dev tenant.
we expect to see for every launch and driver session resource registration call in log:
```
grid_2        | time="2022-12-22T01:16:50.839437971Z" level=info msg="  shape recorded" id=6cb10824f4dc4d7791ee00f73c5a36a2 request body="map[cpu:1024 millicores instant:2022-12-22T01:16:50Z memory:2048 MiB seconds:140.928800199]"
```
P.S. make sure to play with custom Cpu and Memory capabilities to see that customized resources might be tracked as well.

**Comment by vdelendik** [comment]
done: https://github.com/zebrunner/entrypoint/pull/4

1. to test we have to start several api/web jobs and find in esg logs `  shape recorded` messages:
```
docker-compose logs -f grid
grid_1        | time="2022-12-23T11:58:26.261336109Z" level=info msg="    task started" id=0675761562f140e89ca2a3840fc653b3 remote=3.238.101.218 task_definition_family=generic user=qateam
grid_1        | time="2022-12-23T11:58:26.261579325Z" level=info msg="session recorded" id=0675761562f140e89ca2a3840fc653b3 latency=0.133984107 remote=3.238.101.218 user=qateam
grid_2        | time="2022-12-23T12:00:22.989520854Z" level=info msg="    task stopped" id=0675761562f140e89ca2a3840fc653b3
grid_2        | time="2022-12-23T12:00:23.118299926Z" level=info msg="  shape recorded" id=0675761562f140e89ca2a3840fc653b3 request body="map[cpu:1024 millicores instant:2022-12-23T12:00:22Z memory:2048 MiB seconds:116.728370429]"
grid_2        | time="2022-12-23T12:00:23.118334568Z" level=info msg="   task finished" id=0675761562f140e89ca2a3840fc653b3
```

by default any job started on zebrunner executor will use 1024 millicores and 2048 Mib memory.


2. execute web sample job with custom capabilities, for example:
Cpu=2048
Memory=4096
-> in esg grid logs for every browser session (3 tests) we have to record appropriate millicores and MiB of resources.

3. launch web sample test and do an abort via reporting (better to start in 3 threads)
-> all actite web sessions threads (3) and maven executor should report at the end 4 resource usages
